src/backend/dx11/src/lib.rs

```diff
@@ -334,12 +477,97 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
     }
 
     fn format_properties(&self, fmt: Option<format::Format>) -> format::Properties {
-        unimplemented!()
+        let idx = fmt.map(|fmt| fmt as usize).unwrap_or(0);
+        self.format_properties[idx]
     }
 
+    fn image_format_properties(&self, format: format::Format, dimensions: u8, tiling: image::Tiling, usage: image::Usage, storage_flags: image::StorageFlags) -> Option<image::FormatProperties> {
+        conv::map_format(format)?; //filter out unknown formats
 
-    fn image_format_properties(&self, _format: format::Format, dimensions: u8, tiling: image::Tiling, usage: image::Usage, storage_flags: image::StorageFlags) -> Option<image::FormatProperties> {
-        unimplemented!()
+        let supported_usage = {
+            use hal::image::Usage as U;
+            let format_props = &self.format_properties[format as usize];
+            let props = match tiling {
```

Comment:
we should just `impl Index<Tiling> for `format::Properties` and use it in everywhere